src/app/shared/services/auth.service.ts

```diff
@@ -1,70 +1,72 @@
-import {Injectable} from '@angular/core';
-import {environment} from "../../environment";
-import {HttpClient} from "@angular/common/http";
-import {Login} from "../interfaces/login";
-import {Router} from "@angular/router";
-import {Register} from "../interfaces/register";
-import {BehaviorSubject, catchError, Observable, of, Subject} from 'rxjs';
-import {CreateProjectOwner} from "../interfaces/create-project-owner";
-import {CreateDeveloper} from "../interfaces/create-developer";
-import {TokenResponse} from "../interfaces/token-response";
-import {RefreshToken} from "../interfaces/refresh-token";
+import { Injectable } from '@angular/core';
+import { environment } from '../../environment';
+import { HttpClient } from '@angular/common/http';
+import { Login } from '../interfaces/login';
+import { Router } from '@angular/router';
+import { Register } from '../interfaces/register';
+import { BehaviorSubject, catchError, Observable, of, Subject } from 'rxjs';
+import { CreateProjectOwner } from '../interfaces/create-project-owner';
+import { CreateDeveloper } from '../interfaces/create-developer';
+import { TokenResponse } from '../interfaces/token-response';
+import { RefreshToken } from '../interfaces/refresh-token';
 
 @Injectable({
-  providedIn: 'root'
+  providedIn: 'root',
 })
 export class AuthService {
   private apiUrlSecurity: string = `${environment.securityNgRockUrl}/User`;
   private apiUrl: string = `${environment.apiUrl}/ProjectOwner`;
   private apiUrlDev: string = `${environment.apiUrl}/Developer`;
   private apiUrlUser: string = `${environment.apiUrl}/User`;
   isCreated: Subject<boolean> = new Subject<boolean>();
-  private isProjectOwnerSubject = new BehaviorSubject<string>('Default role')
+  private isProjectOwnerSubject = new BehaviorSubject<string>('Default role');
   isProjectOwner$ = this.isProjectOwnerSubject.asObservable();
-  constructor(private httpClient: HttpClient, private router: Router) {
-  }
+  constructor(private httpClient: HttpClient, private router: Router) {}
 
   refreshToken(refreshTokenObj: RefreshToken) {
-    return this.httpClient.post(this.apiUrlSecurity + '/RefreshToken', refreshTokenObj)
+    //TODO: just as an advice, it's common to name routes as 'refresh-token' rather than 'RefreshToken'
```

Comment:
that works of course, but a commonly accepted standard for API naming is 'refresh-token' rather than 'RefreshToken'

this is applicable to other routes